src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -231,7 +233,39 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         ''' </summary>
         Private Sub NeutralLanguageComboBox_DropDown(sender As Object, e As EventArgs) Handles NeutralLanguageComboBox.DropDown
             PopulateNeutralLanguageComboBox(NeutralLanguageComboBox)
-            Common.SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+            SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+        End Sub
+
+        Private Sub LicenseBrowseButton_Click(sender As Object, e As EventArgs) Handles LicenseBrowseButton.Click
+            Dim sInitialDirectory As String = Nothing
+            Dim sFileName As String
+
+            If sInitialDirectory = "" Then
```

Comment:
Since it's a local, `sInitialDirectory` will always be `Nothing` here.

Reply:
Correct - and the whole if else block can be removed since the else block will never be reached...